crates/json-abi/src/abi.rs

```diff
@@ -8,6 +12,7 @@ use serde::{
     ser::SerializeSeq,
     Deserialize, Deserializer, Serialize,
 };
+use std::{io, iter::Flatten};
```

Comment:
`iter` should be used as `core::iter`. 

Any usage of `std` should be feature-wrapped

Reply:
How about `io`, can we still import from `std` @prestwich